src/Symfony/Component/Config/Resource/GlobResource.php

```diff
@@ -52,23 +52,23 @@ public function __construct(?string $prefix, string $pattern, bool $recursive, b
         }
     }
 
-    public function getPrefix()
+    public function getPrefix(): ?string
```

Comment:
This cannot return null. You we're probably irritated by the constructor type. But that is wrong as well. Fixed in #32066